### PR TITLE
feat: Raster from Vector Tiles

### PR DESCRIPTION
**Pull request by jashanbhullar**
- [x] Ported the code takram repo 
- [x] Skip the caching module 
- [x] Fix the build - getting bad import error

### CODE REVIEW

File: geo/tsconfig.json [hunk 1]
```diff
@@ -10,14 +10,20 @@
     "sourceMap": true,
     "outDir": "./dist",
     "baseUrl": "./",
+    "lib": ["es2021", "dom"],
     "incremental": true,
     "skipLibCheck": true,
     "strict": true,
     "strictNullChecks": false,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```
Comment (keiya01) [inline geo/tsconfig.json:18]: We should keep this option being true. I know ported code is working with different rule, but using this option help us in the future.

File: geo/tsconfig.json [hunk 2]
```diff
@@ -10,14 +10,20 @@
...
     "strictBindCallApply": false,
     "forceConsistentCasingInFileNames": false,
     "noFallthroughCasesInSwitch": false,
     "resolveJsonModule": true,
+    "esModuleInterop": true,
+    // TODO: This is not working as of now
```
Comment (keiya01) [inline geo/tsconfig.json:24]: Can remove this comment?
```suggestion
```

**Review by keiya01 — APPROVED** [review]

File: geo/.vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (keiya01) [inline geo/.vscode/settings.json:1]: Is this file necessary?

Comment (jashanbhullar) [reply geo/.vscode/settings.json]: No, I have removed it
